### PR TITLE
[Airflow] Revert metrics field definition to the format used before i…

### PR DESCRIPTION
**Pull request by ritalwar**
- Enhancement

## What does this PR do?

Revert airflow metrics field definition to the format used before introducing metric_type.
This is done because of this [issue](https://github.com/elastic/kibana/issues/155004#issuecomment-1668190898.).
Although, Airflow remained unaffected by this issue. However, with the fix now accessible and the necessary changes present in the elastic-package, it's good to revert to the original format.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #6729 

## Screenshots

Screenshot 2023-08-22 at 12 07 30 PM

Screenshot 2023-08-21 at 2 10 11 PM
Screenshot 2023-08-21 at 2 04 12 PM
Screenshot 2023-08-21 at 2 04 02 PM

### CODE REVIEW

**Comment by agithomas** [comment]
Why the reverting is done? Can you link to the related issue?

**Comment by ritalwar** [comment]
> Why the reverting is done? Can you link to the related issue?

This is the https://github.com/elastic/kibana/issues/155004#issuecomment-1668190898.
Airflow remained unaffected by this issue. However, with the fix now accessible and the necessary changes present in the elastic-package, it's good to revert to the original format.

**Review by tetianakravchenko — APPROVED** [review]

**Review by agithomas — APPROVED** [review]
LGTM!

File: packages/airflow/kibana/dashboard/airflow-a3aa42d0-a465-11ed-9ff0-ab4dd59e4c75.json [hunk 1]
```diff
@@ -435,7 +435,7 @@
                                                     "dataType": "number",
                                                     "isBucketed": false,
                                                     "label": "Scheduler Heartbeat",
-                                                    "operationType": "sum",
+                                                    "operationType": "max",
```
Comment (lalit-satapathy) [inline packages/airflow/kibana/dashboard/airflow-a3aa42d0-a465-11ed-9ff0-ab4dd59e4c75.json:438]: @ritalwar, Was this an aggregation change  to fix some issue?

Comment (ritalwar) [reply packages/airflow/kibana/dashboard/airflow-a3aa42d0-a465-11ed-9ff0-ab4dd59e4c75.json:438]: This was intended for the dashboard, as the sum aggregator isn't compatible with counters.

Comment (agithomas) [reply packages/airflow/kibana/dashboard/airflow-a3aa42d0-a465-11ed-9ff0-ab4dd59e4c75.json:438]: avg() replaced by max() is understandable. But, sum() replaced by a max(), might change what value is shown by the dashboard panel. Can you re-assess the correctness of the dashboard value following this change?

Comment (agithomas) [reply packages/airflow/kibana/dashboard/airflow-a3aa42d0-a465-11ed-9ff0-ab4dd59e4c75.json:438]: Please create an issue for tracking purpose, if needed.

Comment (lalit-satapathy) [reply packages/airflow/kibana/dashboard/airflow-a3aa42d0-a465-11ed-9ff0-ab4dd59e4c75.json:438]: Agree, we can summarise the change in a top level issue (existing Airlfow TSDB issue also works)

Comment (ritalwar) [reply packages/airflow/kibana/dashboard/airflow-a3aa42d0-a465-11ed-9ff0-ab4dd59e4c75.json:438]: Created this [issue](https://github.com/elastic/integrations/issues/7504).

**Pull request merged**